crates/aptos-protos/proto/aptos/remote_executor/v1/network_msg.proto

```diff
@@ -0,0 +1,20 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.remote_executor.v1;
+
+message NetworkMessage {
```

Comment:
why do we choose to use generic message here instead of individual request message type for each rpc?

Reply:
Good question. 
Individual request message type for each rpc is the goal. This is just an initial commit to (i) start evaluating grpc (ii) quickly have a reliable communication version of what we have today.